lib/appsignal/plug.ex

```diff
@@ -99,7 +99,14 @@ if Appsignal.plug? do
 
     @header_keys ~w(
       accept accept-charset accept-encoding accept-language cache-control
-      connection user-agent from referer range
+      connection content-length user-agent from negotiate pragma referer range
+
+      auth-type gateway-interface path-translated remote-host remote-ident
+      remote-user remote-addr request-method server-name server-port
+      server-protocol request-uri path-info client-ip range
+
+      x-request-start x-queue-start x-queue-time x-heroku-queue-wait-time
+      x-application-start x-forwarded-for x-auth-token
```

Comment:
The only one I'm not sure about is `x-auth-token` (same for the Ruby gem) that would undo the purpose of this PR, to filter out sensitive data such as API keys.

What's the reason that's in the Ruby gem, @matsimitsu @thijsc ? And can we leave it out here and in the Ruby gem? If our users want to save this data on a transaction I would say our users should set this manually with a tag/metadata.